src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,129 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.util.ArrayList;
+
+/**
+ * Ui class handles all user interactions of Duke.
+ *
+ * @author Cheong Yee Ming
+ * @version Duke Level-9
+ */
+public class Ui {
+    private static final String LINE = "    ____________________________________________________________";
+    private static final String INDENTATION = "     ";
+
+    /**
+     * Logo of Duke
+     */
+    private static final String GREETING = "Hello from\n"
+            + " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    /**
+     * Message to greet the user.
+     */
+    public void greet() {
+        System.out.println(GREETING);
+        printWithFormat("Hello! I'm Duke\n" + INDENTATION + "What can I do for you?");
+    }
+
+    /**
+     * Message to notify users that a task has been
+     * added successfully into the task list.
+     *
+     * @param task Task to be added into task list.
+     * @param taskLeft Total number of tasks in the task list.
+     */
+    public void taskAddedMessage(Task task, int taskLeft) {
```

Comment:
Perhaps this method name can be changed to a verb (i.e. `printTaskAddedMessage`)?
Similarly for some of the methods below.